### PR TITLE
Add logs to the startup process

### PR DESCRIPTION
**Pull request by czs007**
Signed-off-by: zhenshan.cao <zhenshan.cao@zilliz.com>

### CODE REVIEW

**Review by xige-16 — APPROVED** [review]

**Review by neza2017 — APPROVED** [review]

**Comment by czs007** [comment]
The signature is messed up, close it first, then reopen it later

File: internal/dataservice/server.go [hunk 1]
```diff
@@ -143,14 +145,17 @@ func (s *Server) Start() error {
...
 		return err
 	}
 	if err = s.initMsgProducer(); err != nil {
+		log.Debug("DataService initMsgProducer failed", zap.Error(err))
 		return err
 	}
 	s.startServerLoop()
 	s.UpdateStateCode(internalpb.StateCode_Healthy)
 	log.Debug("start success")
+	log.Debug("DataService", zap.Any("State", internalpb.StateCode_Healthy))
```
[the remark this answers is not in the document]
Comment (czs007) [reply internal/dataservice/server.go:158]: LGTM

**Pull request closed without merging**